### PR TITLE
Update to Elm 0.10

### PR DESCRIPTION
**Pull request by rudolf-adamkovic**

### CODE REVIEW

**Comment by nasakinmaxim** [comment]
In review.

File: AudioBar/AudioBarTests.swift [hunk 1]
```diff
@@ -185,13 +185,15 @@ class AudioBarTests: XCTestCase, Tests {
...
+        expect(update?.commands[0], .player(.setCurrentTime(60)))
+        expect(update?.commands[1], .player(.pause))
     }
 
     func testSeekForwardNearEndWhenPlaying2() {
         let update = expectUpdate(for: .seekForward, model: .readyToPlay(.init(isPlaying: true, duration: 60, currentTime: 58)))
         expect(update?.model, .readyToPlay(.init(isPlaying: false, duration: 60, currentTime: 60)))
-        expect(update?.commands, [.player(.setCurrentTime(60)), .player(.pause)])
+        expect(update?.commands[0], .player(.setCurrentTime(60)))
+        expect(update?.commands[1], .player(.pause))
```
Comment (nasakinmaxim) [inline AudioBar/AudioBarTests.swift:196]: Consider testing `commands.count`.

File: AudioBar/AudioBarTests.swift [hunk 2]
```diff
@@ -185,13 +185,15 @@ class AudioBarTests: XCTestCase, Tests {
     func testSeekForwardNearEndWhenPlaying1() {
         let update = expectUpdate(for: .seekForward, model: .readyToPlay(.init(isPlaying: true, duration: 60, currentTime: 59)))
         expect(update?.model, .readyToPlay(.init(isPlaying: false, duration: 60, currentTime: 60)))
-        expect(update?.commands, [.player(.setCurrentTime(60)), .player(.pause)])
+        expect(update?.commands[0], .player(.setCurrentTime(60)))
+        expect(update?.commands[1], .player(.pause))
```
Comment (nasakinmaxim) [inline AudioBar/AudioBarTests.swift:189]: Consider testing `commands.count`.

File: AudioBar/AudioBarTests.swift [hunk 3]
```diff
@@ -63,7 +63,7 @@ class AudioBarTests: XCTestCase, Tests {
     func testPlayerDidFailToBecomeReady() {
         let update = expectUpdate(for: .playerDidFailToBecomeReady, model: .waitingForPlayerToBecomeReadyToPlayURL(URL.arbitrary))
         expect(update?.model, .readyToLoadURL(URL.arbitrary))
-        expect(update?.commands, [.showAlert(text: "Unable to load media", button: "OK")])
+        expect(update?.command, .showAlert(text: "Unable to load media", button: "OK"))
```
Comment (nasakinmaxim) [inline AudioBar/AudioBarTests.swift]: Consider testing `commands.count`.

**Review by nasakinmaxim — CHANGES_REQUESTED** [review]

**Comment by rudolf-adamkovic** [comment]
@nasakinmaxim 

> Consider testing `commands.count`.

From today's discussion with @andris-zalitis:

@andris-zalitis: true, but now you are still missing one test
@andris-zalitis: that no other commands have been scheduled
@salutis: Number of commands?
@salutis: Ha.
@andris-zalitis: yes
@salutis: This is deeply philosophical.
@salutis: No, it’s not missing.
@andris-zalitis: don’t care about it?
@salutis: You emit a command by calling a function, right?
@salutis: We don’t test functions that ARE NOT called.
@salutis: Same as testing the model.
@andris-zalitis: I agree that it’s philosophical
@salutis: You just check one property. What if something else changed? Well, you don’t test that.
@salutis: Before, I compared the entire model to model.
@salutis: Remember?
@salutis: Also commands to commands.
@salutis: The fact is that it doesn’t scale.
@salutis: Many things depend on many other things in real life.
@andris-zalitis: I see
@salutis: So, I had to configure million irrelevant things to test something.
@salutis: Just to get the entire model match.
@andris-zalitis: as long as we do TDD we should be good
@salutis: Exactly. This is 100% designed for TDD, by the way.

**Review by nasakinmaxim — APPROVED** [review]

**Comment by nasakinmaxim** [comment]
@salutis
>From today's discussion with @andris-zalitis: [...]

Got it. Waiting for CI.

**Pull request merged**